Detectors/TOF/base/include/TOFBase/WindowFiller.h

```diff
@@ -29,9 +29,11 @@ class WindowFiller
 
   void initObj();
 
+  void reset();
+
   Int_t getCurrentReadoutWindow() const { return mReadoutWindowCurrent; }
   void setCurrentReadoutWindow(Double_t value) { mReadoutWindowCurrent = value; }
-  void setEventTime(double value) { mEventTime = value; }
+  void setEventTime(double value) { mEventTime = value - mTF * o2::constants::lhc::LHCOrbitNS * 256; }
```

Comment:
Hi Ruben, two questions:

1) Can you/we set a common variable in Steer or in Common to manage it in a single point for all detectors? I guess the TF choice doesn' affect only tof but to all.

I was using a hardcoded value waiting for that in the common framework.


2) Should be defined as constant or as a setable variable?

Francesco

Reply:
At the moment you can use e.g. "HBFUtils::getNOrbitsPerTF()"  (note, it is not a static method, so you should instantiate it. This does not guarantee that other device will not use HBFUtils with different NOrbitsPerTF, but I think I can manage to change it, so if some workflow modifies it from the command line, it will propagate everywhere).
It should be a parameter, not a constant, and in the MC we will certainly exercise shorter TFs to avoid memory problems.

Reply:
Very good. I will fix everywhere linking HBFUtils. Thanks Francesco

Reply:
DONE (visible soon in the PR)